src/components/search/SearchPage.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { fetchArtists } from '../../services/searchApi';
+import ArtistsList from '../artists/ArtistsList';
+import Pagination from '../pagination/Pagination';
+import SearchForm from './SearchForm';
+
+
+const SearchPage = () => {
+  const [search, setSearch] = useState('');
+  const [artists, setArtists] = useState([]);
+  const [page, setPage] = useState(1);
+
+  const handleInputChange = ({ target }) => {
+    setSearch(target.value);
+  }
+
+  const handleFormSubmit = async (e) => {
+    e.preventDefault();
+    const artists = await fetchArtists(search, page);
+    setArtists(artists);
+  }
+
+  const handleNextClick = async () => {
+    setPage((prevPage) => {
+      return prevPage + 1
+    })
+    const artists = await fetchArtists(search, page);
```

Comment:
lines 27-29 (and also 35 and 36) should be a in  useEffect. Making a fetch call? think useEffect. Need it to change when a page changes? useEffect with a [pageState].